client-v2/src/shared/components/CollectionHoverItems.tsx

```diff
@@ -6,25 +6,10 @@ const HoverActionsAreaOverlay = styled('div')<{
 }>`
   display: ${({ disabled }) => (disabled ? 'none' : 'flex')};
   justify-content: ${({ justify }) => justify};
-  padding: 4px 8px;
 `;
 
 HoverActionsAreaOverlay.defaultProps = {
   justify: 'space-between'
 };
 
-const HideMetaDataOnToolTipDisplay = styled('div')<{
```

Comment:
I'm pretty sure this is redundant, and it looked quite odd on snaplinks in the clipboard, so I chopped it -- double ✂️ -- any problems let me know!